packages/simple-types/src/helpers/notebook-helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { generateEmptyCreationResponse } from "./generate-empty-creation-response";
+import { convertItemToTemplate } from "./convert-item-to-template";
+import { createItemFromTemplate } from "./create-item-from-template";
+import { shareTemplatesToGroups } from "./share-templates-to-groups";
+import { updateNotebookData } from "./update-notebook-data";
+
+// Export the Notebook Helper module
+export {
+  createItemFromTemplate,
+  convertItemToTemplate,
```

Comment:
re-exporting these two functions was needed to get `spyOn` working smoothly, but I think we can resolve this my renaming the functions so they don't exactly match the fn names on the processors. Additionally, once we hoist out the notebook stuff, this becomes less of an issue, as these fn's will really be just for the "simple types"